crates/curp/src/client/stream.rs

```diff
@@ -63,7 +63,7 @@ impl Streaming {
     pub(super) async fn keep_heartbeat(&self) {
         /// Prevent lock contention when leader crashed or some unknown errors
         const RETRY_DELAY: Duration = Duration::from_millis(100);
-        #[allow(clippy::ignored_unit_patterns)]
+        #[allow(clippy::ignored_unit_patterns)] // tokio select internal triggered
```

Comment:
consider enabling `clippy::allow_attributes_without_reason`?

Reply:
This requires `lint_reason` feature. But it seems not stable yet. FYI: https://github.com/rust-lang/rust/issues/54503